src/components/tables/PoolsTable/PoolsTable.vue

```diff
@@ -263,7 +263,8 @@ function aprLabelFor(pool: Pool): string {
   const poolAPRs = pool?.apr;
   if (!poolAPRs) return '0';
 
-  return totalAprLabel(poolAPRs, pool.boost);
+  const boost = pool.boost || props?.boosts?.[pool.id];
```

Comment:
What about using that `boostFor` function above? And shouldn't it also check `pool.boost`?

Reply:
yeah, makes sense